javascript/javascript1/week2/HM.js

```diff
@@ -0,0 +1,96 @@
+// Flight booking fullname function
+
+function getFullname(firstname, surname, userFormatName, sex) {
+  if (userFormatName == true && sex == 'male') {
+    console.log('Lord ' + firstname + ' ' + surname)
+  } else if (userFormatName == true && sex == 'female') {
+    console.log('Ledi ' + firstname + ' ' + surname)
+  } else {
+    console.log(firstname + ' ' + surname)
+  }
+}
+
+const fullname1 = getFullname('Benjamin', 'Hughes', false, '')
+const fullname2 = getFullname('Mac', 'Donalds', true, 'male')
+const fullname3 = getFullname('Barbara', 'Streisand', true, 'female')
+
+// Event application
+
+const Date = [
+  'Monday',
+  'Tuesday',
+  'Wednesday',
+  'Thursday',
+  'Friday',
+  'Saturday',
+  'Sunday',
+]
+let today = 1
+function getNewDate(dayAfter) {
+  let i = today + (dayAfter % Date.length)
+  return Date[i]
+}
+console.log(getNewDate(150))
+
+// Weather wear
+function youClothesToWear(tempr) {
+  if (tempr >= 30 && tempr <= -10) {
+    console.log('Don`t go there')
+  } else if (tempr < 30 && tempr >= 10) {
+    console.log('shorts and a t-shirt')
+  } else if (tempr < 10 && tempr >= 0) {
+    console.log("Don't forget your hat")
+  } else if (tempr < 0 && tempr > -10) {
+    console.log('Are you sure you need to go there?')
+  }
+}
+youClothesToWear(-12)
+// Student manager
+const class07Students = []
+
+function addStudentToClass(studentName) {
```

Comment:
It seems that there's a bug here: there's a way to add same student two times (except the first student). Example:
```
> addStudentToClass('a')
> addStudentToClass('b')
> addStudentToClass('b')
> class07Students
[ 'a', 'b', 'b' ]
```

I'd propose to take another approach and to use [includes](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes) method to detect if student is already in array